WordPress/Classes/ViewRelated/Post/Preview/PreviewWebKitViewController.swift

```diff
@@ -104,6 +104,17 @@ class PreviewWebKitViewController: WebKitViewController {
             showNoResults(withTitle: Constants.noPreviewTitle)
         }
         setupDeviceLabel()
+
+        NotificationCenter.default.addObserver(self, selector: #selector(orientatedChanged), name: UIDevice.orientationDidChangeNotification, object: nil)
+    }
+
+    deinit {
```

Comment:
I _think_ this [isn't required](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver) but it's one less thing for the system to clean up if we're explicit. 👍 

> If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function. If you forget or are unable to remove an observer, the system cleans up the next time it would have posted to it.